schema.sql

```diff
@@ -10,4 +10,19 @@ CREATE TABLE animals (
 );
 
 -- Adding a new column to the animals table
-ALTER TABLE animals ADD COLUMN species VARCHAR(100);
```

Comment:
- [ ] Instead of removing the line, kindly write a query to remove the species column here.